src/modules/importer.js

```diff
@@ -0,0 +1,120 @@
+import fs from 'fs';
+
+export default class Importer {
+  listen(eventEmitter, path, sync = false) {
+    this.eventEmitter = eventEmitter;
+
+    this.eventEmitter.on('changed', () => {
+      if (sync) {
+        console.log(this.importSync(path));
+      } else {
+        let result = this.import(path);
+
+        result.then(data => {
+          console.log(data);
+        });
+      }
+    });
+  }
+
+  import(path) {
+    return new Promise((resolve, reject) => {
+      fs.readdir(path, (err, files) => {

```

Comment:
Please promisify by example above

Reply:
In task: 'import(path): should return a promise with imported data'
If I promisify this method, how can I write this function? I don't see solution with promisify...